depends/packages/native_clang.mk

```diff
@@ -1,19 +1,19 @@
 package=native_clang
-$(package)_major_version=11
-$(package)_version=11.0.1
+$(package)_major_version=12
+$(package)_version=12.0.1
 $(package)_download_path=https://github.com/llvm/llvm-project/releases/download/llvmorg-$($(package)_version)
 $(package)_download_path_linux=https://github.com/llvm/llvm-project/releases/download/llvmorg-$($(package)_version)
 $(package)_download_file_linux=clang+llvm-$($(package)_version)-x86_64-linux-gnu-ubuntu-16.04.tar.xz
 $(package)_file_name_linux=clang-llvm-$($(package)_version)-x86_64-linux-gnu-ubuntu-16.04.tar.xz
-$(package)_sha256_hash_linux=67f18660231d7dd09dc93502f712613247b7b4395e6f48c11226629b250b53c5
-$(package)_download_path_darwin=https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0
-$(package)_download_file_darwin=clang+llvm-11.0.0-x86_64-apple-darwin.tar.xz
-$(package)_file_name_darwin=clang-llvm-11.0.0-x86_64-apple-darwin.tar.xz
-$(package)_sha256_hash_darwin=b93886ab0025cbbdbb08b46e5e403a462b0ce034811c929e96ed66c2b07fe63a
+$(package)_sha256_hash_linux=6b3cc55d3ef413be79785c4dc02828ab3bd6b887872b143e3091692fc6acefe7
+$(package)_download_path_darwin=https://github.com/llvm/llvm-project/releases/download/llvmorg-$($(package)_major_version).0.0
+$(package)_download_file_darwin=clang+llvm-$($(package)_major_version).0.0-x86_64-apple-darwin.tar.xz
+$(package)_file_name_darwin=clang-llvm-$($(package)_major_version).0.0-x86_64-apple-darwin.tar.xz
```

Comment:
This pattern holds for llvm/clang 11 and 12 (i.e. that x.0.1 is released for Linux but not macOS), but is it a general pattern or a coincidence?

Reply:
If it is indeed a coincidence it's easy to revert this, but as I'm the one who does all these dependency updates, I'm selfishly making my own life easier here :)